utils/wasm-builder/src/wasm_project.rs

```diff
@@ -414,6 +433,15 @@ extern "C" fn metahash() {{
 
         fs::create_dir_all(&self.target_dir).context("Failed to create target directory")?;
 
+        for pre_processor in self.pre_processors.iter() {
```

Comment:
No need to call iter imo
```suggestion
        for pre_processor in &self.pre_processors {
```